packages/apps/composer-app/project.json

```diff
@@ -65,18 +78,6 @@
         "{options.outputPath}"
       ]
     },
-    "bundle": {
-      // Don't cache because this is writing into the output of the app bundle
```

Comment:
😭 I always forget the tooling erases comments in these files

Reply:
I didn't even know nx supported comments in those files

Reply:
Why do people still stick with JSON for configuration instead of much better options like YAML or TOML?

Reply:
ubiquity presumably... we could fix this by using [json5](https://json5.org/) in our tooling though because it won't strip out comments